pkgs/by-name/mu/musl/package.nix

```diff
@@ -94,6 +94,13 @@ stdenv.mkDerivation rec {
       url = "https://www.openwall.com/lists/musl/2025/02/13/1/2";
       hash = "sha256-BiD87k6KTlLr4ep14rUdIZfr2iQkicBYaSTq+p6WBqE=";
     })
+    (fetchurl {
+      # required for systemd user namespacing and oomd to work correctly on musl
+      # drop next release
+      name = "stdio-skip-empty-iovec-when-buffering-is-disabled.patch";
+      url = "https://git.musl-libc.org/cgit/musl/patch/?id=fde29c04adbab9d5b081bf6717b5458188647f1c";
```

Comment:
Dynamically generated patch, so shouldn't be fetched with fetchurl.

Reply:
How is this fetch any different than e.g. https://github.com/NixOS/nixpkgs/blob/4afcbefac7c4534b6c74ac53b99addfc63405073/pkgs/development/interpreters/guile/3.0.nix#L94 ? I am not super familiar with cgit, how do i do i obtain a stable link if not this?

Reply:
That link is, correctly, fetchpatched.  fetchpatch normalizes patches to remove stuff like the cgit version at the bottom, which would otherwise break our hashes when cgit is updated.  Here, you are using fetchurl, which does not do that.